reference/price_lists.v3.yml

```
@@ -221,6 +221,38 @@ paths:
                               The total number of pages in the collection.
                             example: 1
                         description: 'Data related to the response, including pagination and collection totals.'
+                      cursor_pagination:
+                        title: Pagination
+                        type: object
+                        properties:
+                          count:
+                            description: Total number of items in the collection response.
+                            type: integer
+                            format: int32
+                          per_page:
+                            description: The amount of items returned in the collection per page, controlled by the limit parameter.
+                            type: integer
+                            format: int32
+                          start_cursor:
+                            description: A string representing the starting point of the current page in the collection
```

```suggestion
                            description: A string representing the starting point of the current page in the collection.
```
🍹  consistency since a lot of the other descriptions end in `.`